tensorrt_llm/_torch/attention_backend/trtllm.py

```diff
@@ -1292,6 +1294,13 @@ def forward(
             # Context MLA uses separate qkv instead of paged_context_fmha
             use_paged_context_fmha = False
 
+        if self.quant_config.layer_quant_mode.has_fp8_kv_cache(
+        ) and self.quant_config.quant_algo == QuantAlgo.W4A16_AWQ and use_paged_context_fmha:
+            logger.warning(
+                "Disabling Paged context FMHA, as it is not supported with FP8 KV cache, for W4A16 AWQ quantization"
+            )
+            use_paged_context_fmha = False
```

Comment:
IMO, I think we should throw out an error and let user disable chunked perfill and kv cache reuse (so paged context FMHA will not be enabled). I am not quite sure if adding a warning is suggested or not.

At the same time, we support FP8 QKV + BF16 output on blackwell, so this cannot be simply disabled. we check if it is on hopper or not.